.github/workflows/Add_PR_and_issues

```diff
@@ -0,0 +1,21 @@
+name: Add To GitHub projects
```

Comment:
this is the action interface and NOT a workflow https://github.com/actions/add-to-project/blob/main/action.yml

Reply:
ok, will make the changes